daemon/cmd/kube_proxy_replacement.go

```diff
@@ -486,16 +486,19 @@ func handleNativeDevices(strict bool) error {
 // finishKubeProxyReplacementInit finishes initialization of kube-proxy
 // replacement after all devices are known.
 func finishKubeProxyReplacementInit(isKubeProxyReplacementStrict bool) error {
```

Comment:
@brb Should we rename this method? It's not really specific to NodePort anymore. Either that or we could move the `InitDeviceAddrs` call outside this function.

Reply:
> we could move the InitDeviceAddrs call outside this function.
:+1: We should refactor / move out other functions from the daemon/cmd/kube_proxy_replacement.go, as they no longer are KPR-only.

Reply:
Done in a new commit.